src/redux/search/actionCreater.js

```diff
@@ -0,0 +1,49 @@
+import { API_KEY, API_URL } from 'react-native-dotenv';
+
+export const SEARCH_START = 'search_start';
+export const SEARCH_SUCCESS = 'search_success';
+export const SEARCH_FAILED = 'search_failed';
+
+export function searchStart() {
+  return {
+    type: SEARCH_START,
+  };
+}
+
+export function searchSuccess(result) {
+  return {
+    type: SEARCH_SUCCESS,
+    payload: result,
+  };
+}
+
+export function searchFailed(error) {
```

Comment:
pls be consequent, if we use arrow function in an other actionCreator then pls use them everywhere.